README.md

```diff
@@ -1,6 +1,33 @@
+# Azure Storage Blob SDK for Go
+The Microsoft Azure Storage SDK for Go allows you to build applications that takes advantage of Azure's scalable cloud storage. 
 
-# Contributing
+This repository contains the open source Blob SDK for Go. Documentation and examples can be found [here](https://godoc.org/github.com/Azure/azure-storage-blob-go/2016-05-31/azblob).
 
+## Features
+* Blob Storage
+	* Create/Read/Update/Delete Block Blobs
+	* Create/Read/Update/Delete Page Blobs
+	* Create/Read/Update/Delete Append Blobs
+
+## Getting Started
+* If you don't already have it, install [the Go Programming runtime](https://golang.org/dl/)
+* Go get the SDK:
+
+```go get -u https://github.com/Azure/azure-storage-blob-go```
+		
+## SDK Architecture
+
+* Storage SDK for Go provides 2 set of APIs: high-level, and low-level APIs
+	* ServiceURI, ContainerURI and BlobURI objects provide the low-level API functionality and maps one-to-one to the [Azure Storage Blob REST APIs](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api)
+	* A set of high-level APIs are provided in highlevel.go file. These functions provide high level abstractions for convenience like uploading a large stream to Blob storage using multiple PutBlock requests
```

Comment:
The high-level APIs provide high-level convenience abstractions such as uploading a large stream to a block blob (using multiple PutBlock requests).